src/Mvc/Mvc.Core/test/ModelBinding/Validation/DefaultObjectValidatorTests.cs

```diff
@@ -1264,11 +1263,12 @@ public void Validate_SuppressValidation_AfterHasReachedMaxErrors_Invalid()
                 });
         }
 
-        [Fact]
-        public void Validate_Throws_IfValidationDepthExceedsMaxDepth()
+        [Theory]
+        [InlineData(1)]
+        [InlineData(5)]
+        public void Validate_Throws_IfValidationDepthExceedsMaxDepth(int maxDepth)
```

Comment:
What's the value of making this a theory test? testing one as an edge case?

Reply:
There were two separate tests before so I just consolidated those in a single one.
I can otherwise remove one and keep just the one with `1`, or revert this change as it is unrelated.

Reply:
This is fine